test/e2e/workload_test.go

```diff
@@ -884,3 +886,325 @@ var _ = Describe("DaemonSet", func() {
 	})
 
 })
+
+var _ = Describe("ReplicationController", func() {
+	BeforeEach(func() {
+		f = root.Invoke()
+	})
+	JustBeforeEach(func() {
+		cred = f.SecretForLocalBackend()
+		if missing, _ := BeZero().Match(cred); missing {
+			Skip("Missing repository credential")
+		}
+		pvc = f.GetPersistentVolumeClaim()
+		err = f.CreatePersistentVolumeClaim(pvc)
+		Expect(err).NotTo(HaveOccurred())
+		repo = f.Repository(cred.Name, pvc.Name)
+
+		backupCfg = f.BackupConfiguration(repo.Name, targetref)
+		rules = []v1beta1.Rule{
+			{
+				Paths: []string{
+					framework.TestSourceDataMountPath,
+				},
+			},
+		}
+		restoreSession = f.RestoreSession(repo.Name, targetref, rules)
+	})
+	AfterEach(func() {
+		err = f.DeleteSecret(cred.ObjectMeta)
+		Expect(err).NotTo(HaveOccurred())
+		err = framework.WaitUntilSecretDeleted(f.KubeClient, cred.ObjectMeta)
+		Expect(err).NotTo(HaveOccurred())
+	})
+	var (
+		testRCBackup = func() {
+			By("Creating ReplicationController " + rc.Name)
+			_, err = f.CreateReplicationController(rc)
+			Expect(err).NotTo(HaveOccurred())
+			err = util.WaitUntilRCReady(f.KubeClient, rc.ObjectMeta)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Creating sample data inside workload")
+			err = f.CreateSampleDataInsideWorkload(rc.ObjectMeta, apis.KindReplicationController)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Reading sample data from /source/data mountPath inside workload")
+			sampleData, err = f.ReadSampleDataFromFromWorkload(rc.ObjectMeta, apis.KindReplicationController)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(sampleData).To(Not(BeEmpty()))
```

Comment:
Use `Expect(sampleData).NotTo(BeEmpty())` instead of `Expect(sampleData).To(Not(BeEmpty()))`